image/cli/mascli/functions/pipeline_config_storage_classes_sno

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# STORAGE_CLASS_RWO
+# -----------------
+# RWO: Usually this role is fulfilled by file storage classes; used by:
+# - cluster monitoring (alert manager)
+# - Db2 (meta and backup volumes)
+#
+# - prometheus_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - prometheus_alertmgr_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - prometheus_userworkload_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - grafana_instance_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - db2_data_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - db2_logs_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - db2_temp_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - kafka_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - mongodb_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - uds_storage_class [ibmc-block-bronze, ocs-storagecluster-ceph-rbd, managed-premium] (doesn't really need to be bronze)
+
+# Unknown/TBC
+# - appconnect_storage_class - doesn't support auto-select storage classes :(
+# - cpd_metadata_storage_class [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+# - cpd_primary_storage_class [ibmc-file-gold-gid, ocs-storagecluster-cephfs, azurefiles-premium]
+# - cpd_service_storage_class [ibmc-file-gold-gid, ocs-storagecluster-cephfs, azurefiles-premium] / WD: [ibmc-block-gold, ocs-storagecluster-ceph-rbd, managed-premium]
+
+function config_pipeline_storage_classes_sno() {
+  if [[ "$SNO_MODE" == "true" ]]; then
```

Comment:
do we need to check it again? this function is called when needs to be called (inside an `if` statement like this)... can we remove that and simplify the code?

Reply:
yes, it is needed. The check is only in this file.

Reply:
I am asking because pipeline_config_storage_classes_sno is called here https://github.com/ibm-mas/cli/pull/106/files#diff-bccb9816831bc1cb7fcffb7c9f3e1d3f893cc1c88ea4e613517eada55c980a1bR130, which is already inside the if statement like the one you are adding here. So, in the end, the file is called when needed and what is inside it don't need to be rechecked... but that is fine, this is a minor point